### ISSUE TITLE
Updates for JP 5.0 ?

### ISSUE THREAD

**Issue by falinuxKr** [dusty-nv/jetson-voice#16]
Any updated containers for JP 5.0.1? 

Thanks

**Comment by apusky** [comment]
same here

**Comment by androiddrew** [comment]
@dusty-nv do you have any plans to update this repo for Jetpack 5.0? Are there newer alternatives to this setup?

**Comment by dusty-nv** [comment]
Its on my TODO list but you could use RIVA instead
________________________________

**Comment by androiddrew** [comment]
Yeah, I have checked out Riva on my Orin dev kit, but I had errors with the inference server.  A concern of mine about using Riva is the terms of service. I read them it closely, but I can't tell if it would allow me to build a product without paying enterprise.

**Comment by mattwilliamson** [comment]
bump. I'd love to use it on the jetson orin nano.

**Comment by hornej** [comment]
would also love to use this on my jetson orin nano!

**Comment by detruby** [comment]
I'd love to use on Jetson Orin Nano running Jetpack 5.1.3.

**Comment by dusty-nv** [comment]
This repo has been superseded by the projects going on a Jetson AI Lab and jetson-containers.  On JetPack 5 you can use Riva ASR/TTS which is much higher quality than the older models in this repo.

**Comment by detruby** [comment]
I got Riva working on my Orin 5.1.3. It is amazing! Even with 2 part disabled (NLP, NMT), it consumes most of the memory. As I am also trying to run the jetson-inference container (probably moving to the new jetson containers), It would be good to reduce the memory consumption a bit more. Thank you very much.
